python/cugraph/centrality/katz_centrality.py

```diff
@@ -46,6 +47,8 @@ def katz_centrality(
         (1/degree_max). Therefore, setting alpha to (1/degree_max) will
         guarantee that it will never exceed alpha_max thus in turn fulfilling
         the requirement for convergence.
+    beta : None
```

Comment:
Should we throw an error if `beta` is passed?

Reply:
currently it is ignored and documented as being ignored.   This is how we treated the other Nx arguments, in BC for example, that we ignore.  We should define a policy for handling not used arguments.

Reply:
added code to raise a  NotImplementedError error